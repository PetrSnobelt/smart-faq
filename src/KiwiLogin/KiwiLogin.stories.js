// @flow

import React from 'react';
import { storiesOf } from '@storybook/react';
import StoryRouter from 'storybook-router';

import KiwiLogin from './';

storiesOf('KiwiLogin', module)
  .addDecorator(StoryRouter())
  .add('Default', () => <KiwiLogin />);
