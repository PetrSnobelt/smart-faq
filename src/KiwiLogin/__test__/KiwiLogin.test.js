// @flow

import * as React from 'react';
import { mount } from 'enzyme';
import configureStore from 'redux-mock-store';
import { MemoryRouter } from 'react-router-dom';
import KiwiLogin from '../';

describe('KiwiLogin', () => {
  const mockStore = configureStore();
  const store = mockStore({});

  const component = mount(
    <MemoryRouter initialEntries={[{ key: 'testKey' }]}>
      <KiwiLogin store={store} />
    </MemoryRouter>,
  );
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
