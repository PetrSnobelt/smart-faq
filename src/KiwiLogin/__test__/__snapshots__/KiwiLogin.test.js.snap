// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KiwiLogin should match snapshot 1`] = `
<MemoryRouter
  initialEntries={
    Array [
      Object {
        "key": "testKey",
      },
    ]
  }
>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "canGo": [Function],
        "createHref": [Function],
        "entries": Array [
          Object {
            "hash": "",
            "key": "testKey",
            "pathname": "/",
            "search": "",
          },
        ],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "index": 0,
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "key": "testKey",
          "pathname": "/",
          "search": "",
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <Connect(WithAuth)
      store={
        Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        }
      }
    >
      <WithAuth
        doSaveToken={[Function]}
        doSignIn={[Function]}
        store={
          Object {
            "clearActions": [Function],
            "dispatch": [Function],
            "getActions": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "subscribe": [Function],
          }
        }
        storeSubscription={
          Subscription {
            "listeners": Object {
              "clear": [Function],
              "get": [Function],
              "notify": [Function],
              "subscribe": [Function],
            },
            "onStateChange": [Function],
            "parentSub": undefined,
            "store": Object {
              "clearActions": [Function],
              "dispatch": [Function],
              "getActions": [Function],
              "getState": [Function],
              "replaceReducer": [Function],
              "subscribe": [Function],
            },
            "unsubscribe": [Function],
          }
        }
      >
        <KiwiLogin
          doSaveToken={[Function]}
          doSignIn={[Function]}
          store={
            Object {
              "clearActions": [Function],
              "dispatch": [Function],
              "getActions": [Function],
              "getState": [Function],
              "replaceReducer": [Function],
              "subscribe": [Function],
            }
          }
          storeSubscription={
            Subscription {
              "listeners": Object {
                "clear": [Function],
                "get": [Function],
                "notify": [Function],
                "subscribe": [Function],
              },
              "onStateChange": [Function],
              "parentSub": undefined,
              "store": Object {
                "clearActions": [Function],
                "dispatch": [Function],
                "getActions": [Function],
                "getState": [Function],
                "replaceReducer": [Function],
                "subscribe": [Function],
              },
              "unsubscribe": [Function],
            }
          }
        >
          <InputText
            name="password"
            onChange={[Function]}
            type="password"
            value=""
          />
        </div>
      </label>
    </form>
    <Link
      replace={false}
      to="/forgotten-password"
    >
      <div
        className="jsx-3302851857 forgot-password"
      >
        <Typography
          type="active"
        >
          Forgot your password?
        </Typography>
      </div>
    </Link>
    <span
      className="jsx-347799831 singIn"
    >
      <Button
        isDisabled={false}
        onClick={[Function]}
        size="large"
        title="Sign In"
        type="primary"
      />
    </span>
  </div>
  <JSXStyle
    css=".KiwiLogin.jsx-347799831{width:480px;padding-top:128px;}div.picture.jsx-347799831 img.jsx-347799831{width:203px;height:156px;}div.picture.jsx-347799831{margin-left:150px;margin-bottom:68px;}div.main.jsx-347799831{margin-left:40px;margin-right:40px;}p.title.jsx-347799831{color:#171b1e;font-size:28px;font-weight:bold;margin-bottom:8px;line-height:1.2;}.forgot-password.jsx-347799831{margin-top:12px;display:inline-block;}.singIn.jsx-347799831{float:right;}form.jsx-347799831{margin-top:32.8px;}div.input.jsx-347799831{width:400px;height:44px;margin-bottom:34px;margin-top:4px;}label.jsx-347799831{width:352px;height:20px;font-size:14px;line-height:1.43;text-align:left;color:#46515e;display:inline-table;}"
    styleId="347799831"
  />
</div>
>>>>>>> create forgotten password page
`;
